### ISSUE TITLE
BRL.Keycode produces bad looking Help-Page 

### ISSUE THREAD

**Issue by MidimasterSoft** [bmx-ng/brl.mod#215]
When I click in BlitzMax-NG at the right  navigationmenu  on **Modules-User Input- Keycodes** I get this wrong formatted help-page:
![grafik](https://user-images.githubusercontent.com/82944421/144419233-b5c11813-723c-48fb-acb3-8dc87184d98d.png)

**Comment by GWRon** [comment]
I would think it is not an issue with brl.mod but with the tools used to generate the help page

Think makedocs is what is used to 
https://github.com/bmx-ng/bmx-ng/blob/master/src/makedocs/makedocs.bmx

if so, then I would ask @woollybah to move the issue to there.

**Comment by MidimasterSoft** [comment]
No, I think they forgot  to transform the keycodes related **index.html** from 2018...
`https://github.com/bmx-ng/brl.mod/blob/master/keycodes.mod/doc/intro.bbdoc`
... into a modern format. 
You can compare this **index.html** with a similar **index.html** from 2019:
`https://github.com/bmx-ng/brl.mod/blob/master/max2d.mod/doc/intro.bbdoc`

Both have (visible in the MaxIde help) a similar table, but in the second the table is written in pure html, but the first has this old-style (markdown?) which still worked in BlitzMax 1.51:


Table in Keycode index.html:

```
<p>
| Mouse Button | Value |
| --- | --- |
| Left mouse button | MOUSE_LEFT |
| Right mouse button | MOUSE_RIGHT |
| Middle mouse button | MOUSE_MIDDLE |
<p>
...
```

tables in newer Max2D index.html:

```
<table>
<tr><th>Drawing command</th><th>Action</th></tr>
<tr><td><a href="#Cls" class="token">Cls</a></td><td>Clears the viewport</td></tr>
<tr><td><a href="#Plot" class="token">Plot</a></td><td>Draws a single pixel</td></tr>
<tr><td><a href="#DrawLine" class="token">DrawLine</a></td><td>Draws a line</td></tr>
...

```
